packages/openapi-v3/src/__tests__/integration/controller-spec.integration.ts

```diff
@@ -462,7 +462,7 @@ describe('controller spec', () => {
       }
 
       const spec = getControllerSpec(MyController);
-      const globalSchemas = (spec.components ?? {}).schemas;
+      const globalSchemas = (spec.components ?? {})?.schemas;
```

Comment:
I'm wondering why this change is suddenly necessary; Did something upstream break this as well?